tests/CMakeLists.txt

```diff
@@ -1,60 +1,104 @@
 # Loader executable
 
-add_executable(manager_call
-    ${TEST_DIR}/manager_caller.c
-    )
-target_include_directories(manager_call PUBLIC ${INCLUDE_DIR})
-target_link_libraries(manager_call PUBLIC chcomp)
+#add_executable(manager_call
```

Comment:
Commented code.

Reply:
Uncommented in https://github.com/capablevms/CHERI-ELF-comp/pull/7/commits/d5e6106c5f6a661414c66aacc2a361cfc93b0f4b.